steam/ext/dota2/client.py

```diff
@@ -0,0 +1,176 @@
+"""Licensed under The MIT License (MIT) - Copyright (c) 2020-present James H-B. See LICENSE"""
+
+from __future__ import annotations
+
+import asyncio
+from functools import partial
+from typing import Final
+
+from ..._const import timeout
+from ..._gc import Client as Client_
+from ...app import DOTA2
+from ...ext import commands
+from ...utils import MISSING
+from .enums import Hero
+from .models import LiveMatch
+from .protobufs.watch import (
+    CMsgClientToGCFindTopSourceTVGames,
+    CMsgGCToClientFindTopSourceTVGamesResponse,
+)
+from .state import GCState  # noqa: TCH001
+
+__all__ = (
+    "Client",
+    "Bot",
+)
+
+
+class Client(Client_):
+    """Represents a client connection that connects to Steam. This class is used to interact with the Steam API, CMs
+    and the Dota 2 Game Coordinator.
+
+    :class:`Client` is a subclass of :class:`steam.Client`, so whatever you can do with :class:`steam.Client` you can
+    do with :class:`Client`.
+    """
+
+    _APP: Final = DOTA2
+    _state: GCState  # type: ignore  # PEP 705
+
+    async def top_live_matches(
+        self,
+        hero: Hero = MISSING,
+        max_matches: int = 100,
```

Comment:
```suggestion
        hero: Hero = MISSING,
        max_matches: int = 100,
```
These should probably be kwarg only